visualization_msgs/srv/GetInteractiveMarkers.srv

```diff
@@ -0,0 +1,12 @@
+---
+# Identifier for the interactive marker server.
+string server_id
```

Comment:
wouldn't this be redundant?

Reply:
If we assume there is one server allowed per service name, then yes. But, considering it's currently possible to have multiple servers with the same name it might be good to keep the ID field to help differentiate them.

Reply:
After settling that we will only expect one server per interactive marker "topic", this is redundant. Removed in 3388b8d